content/departments/engineering/teams/source/index.md

```diff
@@ -8,7 +8,9 @@ The Source team owns everything related to
 - gitserver
 - repo-updater
 
-We provide value to Sourcegraph by ensuring Sourcegraph can reliably index code from any code hosts, with the proper authentication and authorization in place. For more details, read our [Source Department strategy](strategy.md)
+We provide value to Sourcegraph by ensuring Sourcegraph can reliably index code from any code hosts, with the proper authentication and authorization in place.
```

Comment:
```suggestion
We provide value to Sourcegraph by ensuring Sourcegraph can reliably index code from any code hosts, with the proper authentication and authorization in place. We also ensure administrators are properly empowered to configure Sourcegraph to the best of their requirements by creating an awesome admin experience.
```

Reply:
I feel we should talk a bit about the admin experience here since it's also in our purview.

Reply:
Yup, agree with @eseliger here. One liner on the team page is enough I think.